pgxn/neon/walsender_hooks.c

```diff
@@ -0,0 +1,120 @@
+#include "walsender_hooks.h"
```

Comment:
I'd add a comment that it implements XLogReader routine using NeonWALReader.

Reply:
Added